starter-code/server.js

```diff
@@ -56,8 +63,11 @@ app.post('/articles', function(request, response) {
 
   function queryThree(author_id) {
     client.query(
-      ``, // TODO: Write a SQL query to insert the new article using the author_id from our previous query
-      [], // TODO: Add the data from our new article, including the author_id, as data for the SQL query.
+      `INSERT INTO articles (author_id, title, category, "publishedOn", body)
+      VALUES ($1, $2, $3, $4, $5);`, //DONE TODO: Write a SQL query to insert the new article using the author_id from our previous query
+      [
+        request.body.author_id
```

Comment:
need to give values for all the columns, not just the author_id. Hint: where is the data we're sending to the route located?